packages/db-ra-data-rest/test-setup.js

```diff
@@ -0,0 +1,10 @@
+// require('raf/polyfill'); TODO
```

Comment:
Why this TODO? If it's a leftover please remove (or please explain better what we need to do hre)

Reply:
leftover this too!